.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
Can this be removed please?

Reply:
I'll move them to another branch on our fork.